Explorer/src/context/sidebar_context.tsx

```diff
@@ -0,0 +1,94 @@
+import type { PropsWithChildren } from "react";
+import { createContext, useContext, useEffect, useState } from "react";
+
+interface SidebarContextProps {
+  isOpenOnSmallScreens: boolean;
+  isPageWithSidebar: boolean;
+  // eslint-disable-next-line no-unused-vars
+  setOpenOnSmallScreens: (isOpen: boolean) => void;
```

Comment:
```suggestion
  setOpenOnSmallScreens: (/*isOpen: boolean*/) => void;
```

idk if this is better though

Reply:
seems to break the context code